docs/run-nym-nodes/nodes/requester.md

```diff
@@ -198,9 +334,10 @@ If you are adding custom domains, please note that whilst they may appear in the
 
 ### Running an open proxy
 
-If you really, really want to run an open proxy, perhaps for testing purposes for your own use or among a small group of trusted friends, it is possible to do so. You can disable network checks by passing the flag `--open-proxy` flag when you run it. If you run in this configuration, you do so at your own risk.
+If you really, *really* want to run an open proxy, perhaps for testing purposes for your own use or among a small group of trusted friends, it is possible to do so. You can disable network checks by passing the flag `--open-proxy` flag when you run it. If you run in this configuration, you do so at your own risk.
```

Comment:
one really is enough. especially the bold really 😊

Reply:
If you run in this configuration, you do so at your own risk.

Can be changed to 
if you wish to use this configuration...